server/gitrest/packages/gitrest-base/src/utils/redisFs/redis.ts

```diff
@@ -51,8 +51,9 @@ export class Redis {
 		}
 	}
 
-	public async delete(key: string): Promise<boolean> {
-		const result = await this.client.del(key);
+	public async delete(key: string, appendPrefixToKey = true): Promise<boolean> {
+		const keyToDelete = appendPrefixToKey ? this.getKey(key) : key;
```

Comment:
Maybe adding the comment @znewton suggested about why we can sometimes not add prefix would also be helpful for context in the future when we revisit the code